### PR TITLE
Erb to haml conversion

### PR DESCRIPTION
**Pull request by kathrynemary**
This is working as far as I can see.

### CODE REVIEW

File: app/controllers/on_call_times_controller.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@ def index
   end
 
   def new
-    @call_times = OnCallTime.new
+#    @call_times = OnCallTime.new
```
Comment (BintLopez) [inline app/controllers/on_call_times_controller.rb]: If this is no longer necessary, let's delete?

**Comment by BintLopez** [comment]
Merge conflicts :0

File: app/views/devise/mailer/confirmation_instructions.html.haml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+%p
```
Comment (BintLopez) [inline app/views/devise/mailer/confirmation_instructions.html.haml]: Has the corresponding erb file for this one been deleted yet? From the diff, looks like no.

File: app/views/on_call_times/index.html.haml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+%div
```
Comment (BintLopez) [inline app/views/on_call_times/index.html.haml]: Delete the corresponding erb file you're replacing here?

**Comment by kathrynemary** [comment]
It looks like the failing checks are for models not views; I don't think they're failing as a result of this change?

**Comment by BintLopez** [comment]
@kathrynemary -- Don't worry about the test failures. The issue I'm talking about is rendering the volunteers views. Some of them wouldn't render at all while some of them had a wonky display.

**Comment by kathrynemary** [comment]
Ok Imma back up and try to fix this... stay tuned!

**Comment by BintLopez** [comment]
you rock!!!!

**Pull request closed without merging**